.github/workflows/pr_check.yml

```diff
@@ -20,7 +20,9 @@ jobs:
           with:
             python-version: ${{ matrix.python-version }}
         - name: Install dependencies
-          run: python -m pip install --upgrade "tox>=4.0.9,<5" build
+          run: python -m pip install --upgrade "tox>=4.0.9,<5" build lark
+        - name: Generate parser file as a side effect
+          run: python -m hcl2.parser
```

Comment:
Do we need this change?
I believe, during the tox run this file will be generated automatically
Could you please try to remove it?